.github/ci/after_make_test.sh

```diff
@@ -0,0 +1,3 @@
+# Package 
+make package
+mkdir ../artifacts && mv *.deb ../artifacts/
```

Comment:
don't we need to update the debian package information with ours ?

maybe using those cmake variables: https://decovar.dev/blog/2021/09/23/cmake-cpack-package-deb-apt/